lib/benchee/conversion/memory.ex

```diff
@@ -0,0 +1,197 @@
+defmodule Benchee.Conversion.Memory do
+  @moduledoc """
+  Unit scaling for memory converting from bytes to kilobytes and others.
+  """
+
+  alias Benchee.Conversion.{Format, Scale, Unit}
+
+  @behaviour Scale
+  @behaviour Format
+
+  @bytes_per_kilobyte 1024
+  @bytes_per_megabyte @bytes_per_kilobyte * @bytes_per_kilobyte
+  @bytes_per_gigabyte @bytes_per_megabyte * @bytes_per_kilobyte
+  @bytes_per_terabyte @bytes_per_gigabyte * @bytes_per_kilobyte
+
+  @units %{
+      terabyte:        %Unit{
+                      name:      :terabyte,
```

Comment:
Plus it seems the things within the unit aren't indented, I prefer it more like this:

```elixir
%{
  longest_unit: %Unit{
                  property1: val1,
                  prop2:     val2
                }
}
```

that's also what it hopefully should be like in the other formatters :)

Reply:
is there any tool that could do that for me?